src/container/cli.c

```diff
@@ -1665,11 +1661,19 @@ dc_cont_l2g(daos_handle_t coh, d_iov_t *glob)
 	uuid_copy(cont_glob->dcg_uuid, cont->dc_uuid);
 	uuid_copy(cont_glob->dcg_cont_hdl, cont->dc_cont_hdl);
 	cont_glob->dcg_capas = cont->dc_capas;
-	cont_glob->dcg_csum_type = daos_csummer_get_type(cont->dc_csummer);
+	if (daos_csummer_get_csum(cont->dc_csummer))
+		cont_glob->dcg_csum_type =
+					daos_csummer_get_type(cont->dc_csummer);
+	else
+		cont_glob->dcg_csum_type = 0;
 	cont_glob->dcg_csum_chunksize =
 		daos_csummer_get_chunksize(cont->dc_csummer);
 	cont_glob->dcg_csum_srv_verify =
 		daos_csummer_get_srv_verify(cont->dc_csummer);
+	cont_glob->dcg_dedup = daos_csummer_get_dedup(cont->dc_csummer);
```

Comment:
(style)  'dedup' may be misspelled - perhaps 'dedupe'?
(style)  'dedup' may be misspelled - perhaps 'dedupe'?